src/Tizen.Multimedia.Camera/Camera/CameraCapabilities.cs

```diff
@@ -110,6 +111,13 @@ private bool CheckRangeValid(GetRangeDelegate func)
         /// <since_tizen> 4 </since_tizen>
         public bool IsContinuousCaptureSupported { get; }
 
+        /// <summary>
+        /// Gets the extra preview feature's supported state.
```

Comment:
it's correct, but would sound better: "Gets the extra preview feature availability." or "Gets the extra preview feature support state."

Reply:
I changed it to `Gets the extra preview feature support state` and other similar descriptions are changed also.